### PR TITLE
chore: Automatically merge minor dependencies bumps

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

When `patch` or `minor` updates are raised by @dependabot, merge them automatically if all tests pass.

As the PR titles start with `chore:`, these automatic merges should never trigger a package release.

Uses https://github.com/marketplace/actions/dependabot-auto-merge

## Why?

- Less noise for developers, requiring attention only if tests do not pass.
- No more [bundled updates](https://github.com/guardian/commercial-core/pull/130), which make semantic release notes less useful.

### CODE REVIEW

File: .github/workflows/auto-merge.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      - 'gh-pages'
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (sndrs) [inline .github/workflows/auto-merge.yml:14]: I'm pretty sure this token does not have enough permissions to merge https://github.com/marketplace/actions/dependabot-auto-merge#token-scope

you'd need to make a personal access token, which would then be like you merging them all manually. do you want that responsibility? :)

Comment (mxdvl) [reply .github/workflows/auto-merge.yml:14]: Ah, I see, that’s the same issue as `semantic-release`, then… Could we use a @commercial-dev token?

Comment (sndrs) [reply .github/workflows/auto-merge.yml:14]: I'm pretty sure they're per-user, it's an interesting question though...

**Comment by mxdvl** [comment]
This is not going to be possible just yet…

**Pull request closed without merging**